features/state/processor.feature

```diff
@@ -0,0 +1,32 @@
+Feature: State processor
+  In order to use the API independently of the framework
```

Comment:
```suggestion
  In order to have different resource representations in the request body and in in the response body
```

Reply:
done in commit eada6c0bc00762ec50a0c4a2b9889df3bcbf65ae